From 39ce7349fc3cf4b4eb61822f49f2cd9d3d22aa6e Mon Sep 17 00:00:00 2001
Message-Id: <patch-v11-10.12-39ce7349fc3-20220215T235428Z-avarab@gmail.com>
In-Reply-To: <cover-v11-00.12-00000000000-20220215T235428Z-avarab@gmail.com>
References: <cover-v10-0.9-00000000000-20220204T211534Z-avarab@gmail.com>
	<cover-v11-00.12-00000000000-20220215T235428Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 12 Apr 2021 03:01:00 +0200
Subject: [PATCH v11 10/12] fixup earlier, ./t7810-grep.sh (and maybe
 ./t4014-format-patch.sh?) fails without this

---
 grep.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grep.c b/grep.c
index 951d0faeb93..92d369c2364 100644
--- a/grep.c
+++ b/grep.c
@@ -430,7 +430,7 @@ static void compile_fixed_regexp(struct grep_pat *p, struct grep_opt *opt)
 static void compile_regexp(struct grep_pat *p, struct grep_opt *opt)
 {
 	int err;
-	int regflags = 0;//REG_NEWLINE
+	int regflags = REG_NEWLINE;
 
 	if (opt->pattern_type_option == GREP_PATTERN_TYPE_UNSPECIFIED)
 		opt->pattern_type_option = (opt->extended_regexp_option
-- 
2.35.1.1027.g942e091e858

